### DISCUSSION TITLE
Top Executives

### DISCUSSION THREAD

**Discussion by TimdBui** [alvarobartt/investpy#378] [Q&A] [answered]
Hello Alvaro, do you have a code to get the list of all top executives at a company? It is a section within the 'Profile' section.
If you don't have a code, is there anyway you can show us how to get this list. Thank you so much!

**Comment by alvarobartt** [comment, marked as answer]
Hi @TimdBui, can you please provide me more information on how to retrieve those? As I may try to include that in the next release along with some bug fixes and new features I'm working on.

I need you to send me a sample Investing.com URL that contains that information (or multiple URLs if applicable). Also if you can share a screenshot of the webpage highlighting the information you want to include, that will be perfect!

Thank you for this proposal! :hugs:

**Reply by alvarobartt** [reply]
BTW, if this is a useful enhancement/feature, I may suggest you create an issue rather than a Discussion :+1:

**Reply by alvarobartt** [reply]
This has already been defined as a new issue #379! :+1:

**Comment by TimdBui** [comment]
Hello Alvaro,

Thank you for taking the time to reply to my email.

I did post an issue as you instructed.

Here is the url for the 'Top Executives" . It is within the 'Profile' tab.
https://www.investing.com/equities/microsoft-corp-company-profile

Thank you Alvaro!

On Mon, May 31, 2021 at 3:13 AM Álvaro Bartolomé ***@***.***>
wrote:

> Hi @TimdBui <https://github.com/TimdBui>, can you please provide me more
> information on how to retrieve those? As I may try to include that in the
> next release along with some bug fixes and new features I'm working on.
>
> I need you to send me a sample Investing.com URL that contains that
> information (or multiple URLs if applicable). Also if you can share a
> screenshot of the webpage highlighting the information you want to include,
> that will be perfect!
>
> Thank you for this proposal! 🤗
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/alvarobartt/investpy/discussions/378#discussioncomment-805586>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AIFGZDMFPO3GXJ2LPIZQD43TQMZJDANCNFSM45Y2EMXA>
> .
>


-- 
Tim Bui | ***@***.*** | C: 617-291-3692
